src/Backend/Core/Engine/Header.php

```diff
@@ -184,7 +184,7 @@ public function addJS(
         Priority $priority = null
     ): void {
         $module = $module ?? $this->url->getModule();
-
+        $overwritePath = $overwritePath || strpos($file, 'http') === 0 || strpos($file, '//') === 0;
```

Comment:
Wouldn't it be more failsafe to check if the given path is an url? Because now if you have a valid but non-external path (e.g.: /foo//bar-http.js) it won't be minified.

Reply:
this has been fixed